django/contrib/sitemaps/__init__.py

```diff
@@ -67,6 +67,9 @@ def items(self):
     def location(self, obj):
         return obj.get_absolute_url()
 
+    def get_latest_lastmod(self):
```

Comment:
Might be useful to add some unittests for these methods. I don't think we have any tests that don't go through the full request/response cycle, but it might make testing a bit easier to follow if that's not required for every test.